apps/publish/content/common.py

```diff
@@ -268,6 +271,26 @@ def _validate(self, original, updates):
         if len(validation_errors) > 0:
             raise ValidationError(validation_errors)
 
+    def _raise_if_unpublished_related_items(self, original):
+        archive_service = get_resource_service('archive')
+        publishing_warnings_confirmed = strtobool(request.args.get('publishing_warnings_confirmed') or 'False')
```

Comment:
I think you should also check if request exists, it might not be set when auto publishing something via ingest routing etc:
```python
confirmed = request and strtobool(...)
```

Reply:
also not sure what it should do in this case, if it should publish or not, @karelpetrak ?

Reply:
my bad, this is not about publishing but showing a warning, so in case of auto publishing there is no need to warn

Reply:
How can I know if it's a auto publishing? Do we have different service for this?

Reply:
request will be empty, so you can do
```python
if not request:
    return
```

Reply:
auto publishing via ingest routing should publish it